From bunk@stusta.de Sat Mar  4 04:16:15 2006
Date: Sat, 4 Mar 2006 13:15:31 +0100
From: Adrian Bunk <bunk@stusta.de>
To: gregkh@suse.de
Subject: Kobject: kobject.h: fix a typo
Message-ID: <20060304121531.GU9295@stusta.de>
Content-Disposition: inline

It shouldn't cause real harm, but it hurts my eyes.


Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/kobject.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/include/linux/kobject.h
+++ gregkh-2.6/include/linux/kobject.h
@@ -257,7 +257,7 @@ struct subsys_attribute {
 extern int subsys_create_file(struct subsystem * , struct subsys_attribute *);
 extern void subsys_remove_file(struct subsystem * , struct subsys_attribute *);
 
-#if defined(CONFIG_HOTPLUG) & defined(CONFIG_NET)
+#if defined(CONFIG_HOTPLUG) && defined(CONFIG_NET)
 void kobject_uevent(struct kobject *kobj, enum kobject_action action);
 
 int add_uevent_var(char **envp, int num_envp, int *cur_index,
